routes/index.js

```diff
@@ -1,279 +1,43 @@
+  app.all('/*', (req, res, next) => {
     // todo verify authenticated and reroute if necessary
     // console.log('req.url', req.url)
     next()
   })
 
-  app.get(['/', '/index.html'], (req, res) => {
-    res.sendFile(index)
-  })
-
-  app.get('/websocket.html', (req, res) => {
-    res.sendFile(websocket)
-  })
-
-  app.get('/index2.html', (req, res) => {
-    res.sendFile(index2)
-  })
-
-  app.get('/index3.html', (req, res) => {
-    res.sendFile(index3)
-  })
-
-  app.get('/index4.html', (req, res) => {
-    res.sendFile(index4)
-  })
-
-  app.get('/index5.html', (req, res) => {
-    res.sendFile(index5)
-  })
-
-  app.get('/public/player.js', (req, res) => {
-    res.sendFile(playerJs)
-  })
-
-  app.get('/public/player.min.js', (req, res) => {
-    res.sendFile(playerMinJs)
-  })
-
-  app.get('/public/player.css', (req, res) => {
-    res.sendFile(playerCss)
-  })
-
-  app.param('id', (req, res, next, id) => {
-    // console.log('parse id')
-    const streams = app.locals.streams
-    const stream = streams.get(id)
-    if (!stream) {
-      return res.status(404)
-        .end(`stream "${id}" not found`)
-    }
-    if (!stream.hasOwnProperty('ffmpeg')) {
-      return res.status(500)
-        .end(`stream "${id}" does not have a valid ffmpeg src`)
-    }
-    res.locals.id = id
-    res.locals.stream = stream
-    next()
-  })
+  app.use(express.static('www'))
 
-  app.param('type', (req, res, next, type) => {
-    // console.log('parse type')
-    const stream = res.locals.stream
-    switch (type) {
-      case 'mp4' :
-        if (!stream.ffmpeg.running) {
-          return res.status(503)
-            .end(`stream "${res.locals.id}" is currently not running`)
-        }
-        if (stream.hasOwnProperty('mp4frag')) {
-          res.locals.mp4frag = stream.mp4frag
-          return next()
-        }
-        return res.status(404)
-          .end(`mp4 type not found for stream ${res.locals.id}`)
-      case 'jpeg' :
-        if (!stream.ffmpeg.running) {
-          return res.status(503)
-            .end(`stream "${res.locals.id}" is currently not running`)
-        }
-        if (stream.hasOwnProperty('pipe2jpeg')) {
-          res.locals.pipe2jpeg = stream.pipe2jpeg
-          return next()
-        }
-        return res.status(404)
-          .end(`jpeg type not found for stream ${res.locals.id}`)
-      case 'cmd' :
-        if (stream.hasOwnProperty('ffmpeg')) {
-          res.locals.ffmpeg = stream.ffmpeg
-          return next()
-        }
-        return res.status(404)
-          .end(`cmd type not found for stream ${res.locals.id}`)
-      case 'debug' :
-        return res.end(inspect(res.locals.stream, { sorted: true, showHidden: false, compact: false, depth: 2, colors: false, breakLength: 200, getters: true}))
-      default :
-        res.status(404)
-          .end(`${type} type not found for stream ${res.locals.id}`)
-    }
-  })
+  app.param('id', id)
+  app.param('type', type)
```

Comment:
I split these to their own files, honestly they probably could live in a single file if you wanted.